### DISCUSSION TITLE
Seeing lots of "invalid workflow file" errors all of a sudden … is something going on?

### DISCUSSION THREAD

**Discussion by craiga** [community/community#22387] [New to GitHub]
<p>Hi, I’ve just started seeing a few “invalid workflow file” errors due to a concurrency group with an expression, even when I’ve not touched the workflow file in question for a while.</p>
<p>Is there an issue with GitHub at the moment, or has something here changed?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/4/04065c3594a1850d8d80380ed8c45abb46e384e5.jpeg" data-download-href="/uploads/short-url/zBxgqulw2mrXWzknOmhIgbHWtv.jpeg?dl=1" title="Screenshot 2021-12-03 at 17.01.15" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/76330/181070697-2d798d7f-9255-47db-8c88-69e627733da3.jpeg" alt="Screenshot 2021-12-03 at 17.01.15" data-base62-sha1="zBxgqulw2mrXWzknOmhIgbHWtv" width="594" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04065c3594a1850d8d80380ed8c45abb46e384e5_2_594x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04065c3594a1850d8d80380ed8c45abb46e384e5_2_891x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04065c3594a1850d8d80380ed8c45abb46e384e5_2_1188x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/04065c3594a1850d8d80380ed8c45abb46e384e5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2021-12-03 at 17.01.15</span><span class="informations">2370×1992 416 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<pre><code class="lang-yaml">name: Disaster recovery test
on:
  schedule:
    - cron: "0 10 * * *"
  pull_request:
    paths:
      - ".github/workflows/disaster-recovery-test.yaml"

env:
  HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
  CYPRESS_MAILTRAP_API_TOKEN: ${{ secrets.MAILTRAP_API_TOKEN }}

concurrency:
  group: disaster-recovery-${{ github.ref }}
  cancel-in-progress: true

jobs:
  # …and so on
</code></pre>

**Comment by mudler** [comment, marked as answer]
<p>Hi,</p>
<p>This look similar to <a href="https://github.community/t/breaking-change-to-concurrency-group-syntax/">https://github.community/t/breaking-change-to-concurrency-group-syntax/</a></p>

**Comment by craiga** [comment]
<p>You’re right—thanks!</p>
